e2e/playwright/test-utils.ts

```diff
@@ -0,0 +1,116 @@
+import { expect, Page } from '@playwright/test'
+import { EngineCommand } from '../../src/lang/std/engineConnection'
+
+async function waitForPageLoad(page: Page) {
+  // wait for 'Loading KittyCAD Modeling App...' spinner
+  await page.waitForFunction(() =>
+    document.querySelector('[data-testid="initial-load"]')
```

Comment:
TIL about `data-testid` 😃

Reply:
Oh nice, yeah bit of a convention. Were you familiar with `data-` attributes in general?